src/setup.ts

```diff
@@ -1,12 +1,18 @@
+import { NgModule, provideZonelessChangeDetection } from "@angular/core";
 import { getTestBed } from "@angular/core/testing";
 import {
   BrowserTestingModule,
   platformBrowserTesting,
 } from "@angular/platform-browser/testing";
 
-export function setupAngularTestEnvironment(): void {
+export function setupAngularTestEnvironment(isZoneless = true): void {
```

Comment:
I think it's better to have an "options" object with a "zoneless" boolean instead of a direct variable, makes the call more readable later